cyberdrop_dl/utils/transfer/backup.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+from shutil import copy2
+
+
+def db_backup(db_file):
+    i = 2
+    while True:
+        new_file = Path(f"{db_file}{i}")
+        if new_file.exists():
+            i = i + 1
+            continue
+        else:
+            copy2(db_file, new_file)
+            break
```

Comment:
I suggest using a more descriptive name for the backup, with the current `datetime` as a suffix instead of an index.

Backup should be something like `cyberdrop_v4_YYYYMMDD_HHMMSS.bak.db`